common/reportMetrics.py

```diff
@@ -92,7 +92,7 @@ def get_parameters():
     if options.chunkLines is None and parameters['mode'] == 'metricFileReplay':
         parameters['chunkLines'] = 100
     elif options.chunkLines is None:
-        parameters['chunkLines'] = 40000
```

Comment:
Copying from my jira post:
40k worked in a lot of cases. See II-3239 for how delicate chunk size handling is. Chunks too large take too long to process and get stuck in collection/training/detection. Flood the queue with chunks, and the receiving queue gets overloaded, potentially taking rabbitMQ down due to memory issues.

We should switch to using chunk size (500kb - 1MB?) to take the guesswork out.
* it’s already defined as a CLI option
* it’s already calculated in send_data, so just copy that method